python/scripts/rfi_receiver.py

```diff
@@ -787,6 +789,73 @@ def http_server2():
     httpd.serve_forever()
 
 
+def rfi_zeroing():
+    # Wait until the correct UTC time (deals with daylight savings time)
+    t_now = datetime.utcnow()
+    t_transit = ephemeris.solar_transit(t_now)[0]
+    t_diff = datetime.utcfromtimestamp(t_transit) - t_now
```

Comment:
Shouldn't we be subtracking half the `downtime` from this number?  It looks to me like we stop zeroing exactly at solar transit, not 30 minutes before.